server/models/user.js

```diff
@@ -0,0 +1,17 @@
+var mongoose = require('mongoose');
```

Comment:
```suggestion
const mongoose = require('mongoose');
```
Can you update all the vars to const or let

Reply:
My mistake. Done.